src/Tools/Source/CompilerGeneratorTools/Source/BoundTreeGenerator/BoundNodeClassWriter.cs

```diff
@@ -961,6 +960,26 @@ private void WriteUpdateMethod(Node node)
                         WriteLine("return this;");
                         Unbrace();
                         break;
+
+                        string notEquals(Field field)
```

Comment:
>notEquals [](http://example.com/codeflow?start=31&length=9)

Consider placing this local function where `wasUpdatedCheck` used to be. Perhaps the target language should be included into the name.

Reply:
Factored the two implementations into separate methods `WriteUpdatedMethodCSharp` and `WriteUpdatedMethodVB`